src/routes/users.js

```diff
@@ -0,0 +1,52 @@
+const got = require('got');
+const path = require('path');
+const configImport = require('../config');
+
+const config = configImport.config();
+
+module.exports = {
+    getUsers: async function (req, res) {
+        const groupQuery = `/groups?$filter=startswith(displayName,'dxdeveloper')&$select=displayName,id`;
```

Comment:
This is a quite big function. I'm sure we can refactor into a set of handful little meaningful functions.